apps/token-manager/app/package.json

```diff
@@ -41,13 +41,14 @@
     "prettier": "^1.8.2"
   },
   "scripts": {
-    "test": "jest",
     "lint": "eslint ./src",
-    "start": "npm run sync-assets && npm run watch:script & parcel serve index.html -p 3003 --out-dir build/",
-    "build": "npm run sync-assets && npm run build:script && parcel build index.html --out-dir build/ --public-url \".\"",
-    "build:script": "parcel build src/script.js --out-dir build/",
-    "watch:script": "parcel watch src/script.js --out-dir build/ --no-hmr",
```

Comment:
I decided to remove this script and only leave `watch`.

Reply:
Sounds good!